config/default/manager_auth_proxy_patch.yaml

```diff
@@ -47,3 +47,4 @@ spec:
       - name: manager
         args:
         - "--metrics-addr=127.0.0.1:8080"
+        - "--leader-elect"
```

Comment:
Should be —leader-elect=true ?

Reply:
@yuzisun It's boolean, true is optional.